test/e2e/origins.test.js

```diff
@@ -1,45 +1,46 @@
 import provider from 'eth-provider'
+import { hexToNumber, numberToHex } from 'web3-utils'
 
 jest.mock('../../main/store/persist')
 
 let frame
 
 beforeEach(done => {
-  frame = provider('frame', { origin: 'frame.test' })
+  frame = provider('frame')
```

Comment:
what's the reason for removing this origin?

Reply:
We don't have any unknown origin tests. Can put it back too 🤷

Reply:
it probably doesn't matter much either way but if we're just going to have the one e2e test for now I think it's better to test the more "normal" flow. we can add specific tests around handling unknown origins later if we think we need it